Ref of periodic table

mgallo2019 opened the pull request:
Ver descripciones en los Commits

mgallo2019 committed:
AddButtonRefColors

mgallo2019 committed:
AddPanelRefColors

Incluye:
Imagen con ref colores en panel DRAG que se puede mover por la pantalla, esto se hizo asi porque la grid layout no se puede colocar objetos mas grandes que los botones de la tabla.
El fader se creo 2 metodos nuevos para controlar raycasts de GO. Necesario para mostrar la imagen de ref.
Se agrega clase para efecto de boton para cambiar texto (se usara para la historia de cambiar texto boton tper)
Para el panel DRAG de uso el asset de LEAN GUI (Se hizo prefab)
Ademas se agrego tooltip al nuevo boton referencias colores.

mgallo2019 committed:
AddRefParamElemens

Incluye:
Cambio enla disposicion de los objetos en la grilla de la tabla periodica, quedaron arriba. Y juntos.
Ademas se hizo un fix sobre el metodo que posiciona los tooltips teniendo en cuenta la resolucion porque me di cuenta que quedaban mal.
Se agrego los registros de las comb mas comunes y el script de insert

mgallo2019 commented:
Verif close button de ref param. Creo q olvide modificar el click event

ldgobbi commented on New Unity Project/Assets/Scripts/UI/EffectButtonPress.cs:
```diff
@@ -0,0 +1,27 @@
+    {
+        originalText = myText.text;
+        counter++;
+        if (counter % 2 == 1)
```
Por qué no un `boolean`?
```
original = !original;
if (original)
```

mgallo2019 committed:
Fixes

Incluye: Mejoras indicadas por matias ali. sobre reso de imagenes, posicion de tooltip y botones se cambio el diseño y tamaño de letra

mgallo2019 commented:
.. copy - paste se llama eso... del foro

El vie., 11 oct. 2019 a las 16:24, Leonel Gobbi (<notifications@github.com>)
escribió:

> *@ldgobbi* commented on this pull request.
> ------------------------------
>
> In New Unity Project/Assets/Scripts/UI/EffectButtonPress.cs
> <https://github.com/AliMatias/SIAMM/pull/87#discussion_r334141397>:
>
> > +        counter++;
>
> +        if (counter % 2 == 1)
>
>
> Por qué no un boolean?
>
> original = !original;
>
> if (original)
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AliMatias/SIAMM/pull/87?email_source=notifications&email_token=ALXPQ7YC2JCN2FKU2PG7HNDQODHITA5CNFSM4I7TDN2KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCHXRXTY#pullrequestreview-300882895>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALXPQ74FQIQX6LTP32MLX4TQODHITANCNFSM4I7TDN2A>
> .
>

mgallo2019 committed:
QuitadeScript

Fix: Para ver luego en otro branch el uso del script de efffect press button. Se quita directamente por comentario de otro desarrollador

mgallo2019 commented:
Listo se quita el script con tiempo se ve. Saludos

El sáb., 12 oct. 2019 a las 11:38, Miguel Gallo (<midway2004@gmail.com>)
escribió:

> .. copy - paste se llama eso... del foro
>
> El vie., 11 oct. 2019 a las 16:24, Leonel Gobbi (<notifications@github.com>)
> escribió:
>
>> *@ldgobbi* commented on this pull request.
>> ------------------------------
>>
>> In New Unity Project/Assets/Scripts/UI/EffectButtonPress.cs
>> <https://github.com/AliMatias/SIAMM/pull/87#discussion_r334141397>:
>>
>> > +        counter++;
>>
>> +        if (counter % 2 == 1)
>>
>>
>> Por qué no un boolean?
>>
>> original = !original;
>>
>> if (original)
>>
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/AliMatias/SIAMM/pull/87?email_source=notifications&email_token=ALXPQ7YC2JCN2FKU2PG7HNDQODHITA5CNFSM4I7TDN2KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCHXRXTY#pullrequestreview-300882895>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ALXPQ74FQIQX6LTP32MLX4TQODHITANCNFSM4I7TDN2A>
>> .
>>
>

AliMatias committed:
Merge branch 'develop' into RefOfPeriodicTable
